src/ui/components/UserDefineDescription/index.tsx

```diff
@@ -33,351 +35,290 @@ const UserDefineDescription: FunctionComponent<UserDefineDescriptionProps> =
-                  Check our Wiki page for latest definition.
+                  {t('UserDefineDescription.Wiki')}
                 </DocumentationLink>
               </p>
             </div>
           );
         case UserDefineKey.REGULATORY_DOMAIN_EU_CE_2400:
           return (
             <div>
-              <p>
-                For firmware versions earlier than v3.x, the EU CE 2.4 GHz
-                regulatory domain limits the maximum power to 10mW, which does
-                not require LBT (Listen Before Talk). For firmware versions
-                later than v3.x, the EU CE 2.4 GHz regulatory domain limits the
-                maximum power to 100mW, and enables LBT (Listen Before Talk).
-              </p>
+              <p>{t('UserDefineDescription.RegulatoryDomainEUCE2400')}</p>
               <p>
                 <DocumentationLink url="https://www.expresslrs.org/software/user-defines/#regulatory-domain">
-                  Check our Wiki page for latest definition.
+                  {t('UserDefineDescription.Wiki')}
                 </DocumentationLink>
               </p>
             </div>
           );
         case UserDefineKey.BINDING_PHRASE:
           return (
             <div>
-              <p>
-                Binding phrase pairs receivers to transmitters without requiring
-                you to bind manually. Receivers flashed without a binding phrase
-                will require the traditional binding method. Both the TX and RX
-                need to have the same binding phrase to connect. Set something
-                memorable, and limit to alphanumeric phrases conforming to the
-                Latin alphabet. This is not a password, and does not provide any
-                security, it simply reduces RF collisions with other pilots.
-              </p>
+              <p>{t('UserDefineDescription.BindingPhrase')}</p>
               <p>
                 <DocumentationLink url="https://www.expresslrs.org/software/user-defines/#binding-phrase">
-                  Check our Wiki page for latest definition.
+                  {t('UserDefineDescription.Wiki')}
                 </DocumentationLink>
               </p>
             </div>
           );
         case UserDefineKey.AUTO_WIFI_ON_INTERVAL:
           return (
             <div>
-              <p>
-                This will automatically turn the wifi on for any module that has
-                an ESP8285 on it if no connection is found after set amount of
-                seconds from boot. This enables pushing firmware updates to the
-                RX by connecting to its wifi network and visiting 10.0.0.1
-              </p>
-              <p>
-                Auto on interval is defined in <strong>seconds</strong>. Default
-                is 60 seconds.
-              </p>
+              <Trans i18nKey="UserDefineDescription.AutoWifiOnInterval">
+                <p>
```

Comment:
Similar to the comment about the default property, no need to include the actual text when using the i18nKey property.

Reply:
Fixed.